File: submissions/sm_021_piyush-jain/week_3/day_3/session2/average_divisible_while.js
var limit=20,num=4,average,sum=0,n=0,i=1
while(i<limit)
{
  if((i%num)==0)
  {
      sum=sum+i
      n=n+1
  }
  i++
}

average=sum/n
console.log(average)

File: submissions/sm_107_hasmuddin/week_13/day_4/session_1/average_divisible_while.py
''' Print the average of all the
numbers from 0 to the given limit that are divisible by the given number'''
limit = int(input("limit: "))
num = int(input("Number: "))
sum = 0
count = 0
i = 0
while i < limit:
    if i % num == 0:
        sum += i
        count += 1
    i += 1
ave = sum/count
print(ave)